### ISSUE TITLE
Multiselect accepts non-array initialValue, single select doesn't accept array

### ISSUE THREAD

**Issue by skateman** [data-driven-forms/react-forms#179]
There's a slight inconsistency in how `initialValue` is handled between single and multiselect. Would it be possible for the single select to accept an array of one element?

**Comment by Hyperkid123** [comment]
I think we should do it the other way around honestly. I think its a bug that multi select accepts single value. But i see why you want it this way.

I think we should make this work.

I believe it comes from the react-select library.

**Comment by skateman** [comment]
Thanks :+1:

**Comment by skateman** [comment]
It would help a lot

**Comment by Hyperkid123** [comment]
@skateman I think there might be slight issue with this.

What if you want your value to be an array? For example you have some option where the values are
```jsx
[{
  label: 'Option one',
  value: ['foo']
}, {
  label: 'Option two',
  value: ['foo', 'bar']
}]
```

I think we should hide this obscure use case behind some prop. By default single select would take the first element from an array if it exists. And when you pass some prop it will expect an array.

**Comment by skateman** [comment]
Wait, why would I set multiple values in a multiselect? Or is it the `simpleValue: false` solution? I'm not interested in that one.

**Comment by Hyperkid123** [comment]
That was for single select not multi select. I did see some cases when value was in fact not a literal but an array.

TLDR. It will work for single select as you require. PR is on its way.

**Comment by skateman** [comment]
:+1:
